api/types/servers.go

```diff
@@ -1,16 +1,16 @@
 package types
 
 type Server struct {
-	Id               string `json:"id" header:"ID"`
-	Name             string `json:"name" header:"NAME"`
-	Fqdn             string `json:"fqdn" header:"FQDN"`
-	State            string `json:"state" header:"STATE"`
-	Public_ip        string `json:"public_ip" header:"PUBLIC_IP"`
-	Workspace_id     string `json:"workspace_id" header:"WORKSPACE_ID"`
-	Template_id      string `json:"template_id" header:"TEMPLATE_ID"`
-	Server_plan_id   string `json:"server_plan_id" header:"SERVER_PLAN_ID"`
-	Cloud_account_id string `json:"cloud_account_id" header:"CLOUD_ACCOUNT_ID"`
-	Ssh_profile_id   string `json:"ssh_profile_id" header:"SSH_PROFILE_ID"`
+	Id                  string `json:"id" header:"ID"`
+	Name                string `json:"name" header:"NAME"`
+	Fqdn                string `json:"fqdn" header:"FQDN"`
+	State               string `json:"state" header:"STATE"`
+	Public_ip           string `json:"public_ip" header:"PUBLIC_IP"`
+	Template_id         string `json:"template_id" header:"TEMPLATE_ID"`
+	Server_plan_id      string `json:"server_plan_id" header:"SERVER_PLAN_ID"`
+	Cloud_account_id    string `json:"cloud_account_id" header:"CLOUD_ACCOUNT_ID"`
+	Ssh_profile_id      string `json:"ssh_profile_id" header:"SSH_PROFILE_ID"`
+	Firewall_profile_id string `json:"firewall_profile_id" header:"FIREWALL_PROFILE_ID"`
```

Comment:
We should be using CamelCase instead of snake_case for struct field names to comply with idiomatic Go code. You should refactor the old fields to comply or at least not introduce new fields in snake_case...

Reply:
Sure. I think so too. I'm going to review these things